python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -307,7 +307,7 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
             from Ganga.GPIDev.Base.Proxy import isType, getRuntimeGPIObject, stripProxy, getName
             from Ganga.GPIDev.Base.Objects import Node
             if isinstance(defvalue, Node):
-                return stripProxy(getRuntimeGPIObject(getName(defvalue)))
+                return allPlugins.find(item['category'], getName(defvalue))()
```

Comment:
Can we check that it's not already an instance incase there is an instance stored here, also you would need to take a deepcopy to avoid problems with the default possibly being changed